posix.mak

```diff
@@ -94,7 +94,7 @@ install: $(TOOLS) $(CURL_TOOLS) $(ROOT)/dustmite
 	cp $^ $(INSTALL_DIR)/bin
 
 clean:
-	rm -f $(ROOT)/dustmite $(TOOLS) $(CURL_TOOLS) $(DOC_TOOLS) $(TAGS) *.o $(ROOT)/*.o
+	rm -rf $(ROOT_OF_THEM_ALL)
```

Comment:
Minor note here: I wasn't sure what the `$(TAGS)` stuff was, since it doesn't appear to have any definition within the makefile itself.  Should that be re-added?

Same question applies to the `win32.mak` changes.

Reply:
BTW, `-f` usually isn't needed with `-r`, probably better to just use `rm -R` without `-f`, like the DMD makefile.